config/jobs-async-upload/examples/gen-update-artifact.sh

```
@@ -18,4 +18,5 @@ oc process --local -f jobs-async-upload-uri-to-oci-template.yaml \
   -p MODEL_SYNC_SOURCE_URI="https://github.com/onnx/models/raw/refs/heads/main/validated/vision/classification/mnist/model/mnist-8.onnx" \
   -p MODEL_SYNC_DESTINATION_OCI_URI="default-route-openshift-image-registry.apps.rosa.my-cluster.019m.p3.openshiftapps.com/project3/mnist-v3:latest" \
   -p DESTINATION_CONNECTION=oci-credentials \
+  # -p MODEL_SYNC_DESTINATION_OCI_ENABLE_TLS_VERIFY=false \  # Uncomment for registries with self-signed certs
   -o yaml > job.yaml
```

Shellcheck (0.11.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)